lib/tasks/gather.js

```diff
@@ -321,7 +321,8 @@ class TaskGather extends SttTask {
     this.logger.debug(evt, 'TaskGather:_onDtmf');
     clearTimeout(this.interDigitTimer);
     let resolved = false;
-    if (this.dtmfBargein) {
+    if (this.dtmfBargein && !this.playComplete) {
```

Comment:
this doesnt look correct to me.  Given this code, once the play is complete we stop emitting dtmf events.  I understand you want to send a notify when the first dtmf is detected but you have included other side affects.

Reply:
if condition corrected and also referring the 0.2.9 version of time-series